modules/pel/peltool/peltool.py

```diff
@@ -305,6 +380,15 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        path = "/var/lib/phosphor-logging/extensions/pels/logs/"
+        if not os.path.isdir(path):
+            path = input("Enter folder path: ")
```

Comment:
Oh, I meant check if on a BMC or not before doing the add_argument so you can handle it differently.  anything we do either way needs to be scriptable so can't ask extra questions.